run_knn_client.py

```diff
@@ -0,0 +1,13 @@
+import json
```

Comment:
Can we put these "template" scripts (e.g. ones you aren't expected to run directly, but copy and modify as your own) in a directory? `examples/` or `templates/` or something.

Reply:
Yes!